src/Messages.js

```diff
@@ -120,20 +130,41 @@ class Messages extends React.Component {
     var names = this.state.messageSender;
     return (
       <div>
-        <Link to={{pathname: '/matches', state: {id: this.state.userId}}}>Back to Matches matches</Link>
+        <Header id={this.state.id}/>
+      <div className = "PageContainer">
+        <p>You are currently messaging:</p>
+         <h1>{this.state.friendName}</h1>
+        <div className = "MessageContainer">
         {messages.length > 0 ?
           messages.map((msg, index) => (
-            <div>
-              <p>{timeStamps[index]}</p>
-              <p>{names[index]} : {msg}</p>
-            </div>
-          )):
-          <header>Start a conversation!</header>}
+            //Checks which user is messaging
+             names[index]==this.state.currentName ?
+              <div className = "UserOne">
+                <p1>{timeStamps[index]}</p1>
+                <p>{names[index]} : {msg}</p>
+              </div>
+            :
```

Comment:
can you reformat indentation to make it more clear which ternary statement the `:` belongs to